docs/en/sql-reference/sql-statements/Data Manipulation/SHOW ROUTINE LOAD.md

```diff
@@ -37,13 +37,11 @@ SHOW ROUTINE LOAD FOR test1;
 
 3. Display all routine import jobs (including stopped or cancelled jobs) under example_db. The result is one or more lines.
 
-use example_db;
-SHOW ALL ROUTINE LOAD;
+SHOW ALL ROUTINE LOAD FROM example_db;
```

Comment:
It’s better to use the in keyword as an example. The from keyword has other meanings in some sentences.